src/python/crash_analysis/stack_parsing/stack_symbolizer.py

```diff
@@ -526,7 +526,6 @@ def filter_binary_path(binary_path):
     app_directory = os.path.dirname(app_path)
     binary_filename = os.path.basename(binary_path)
     nfs_directory = environment.get_value('NFS_ROOT')
-    root_directory = environment.get_value('ROOT_DIR')
```

Comment:
I didn't think there's a bug by not using this variable so removing.

Reply:
sorry accidental click